src/person.rb

```diff
@@ -1,16 +1,17 @@
 require './nameable'
 
 class Person
+  attr_reader :id, :rental
+  attr_accessor :name, :age
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
-  attr_reader :id
-  attr_accessor :name, :age
-
   private
 
   def of_age?
```

Comment:
- [ ] It will be great to have a method in your person module which is responsible for adding a new rental. Remember a person has many books through rentals. And the book has many person through the rentals. So you should have a method that looks like this ⬇️ 
```
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```